src/main/java/com/minecolonies/coremod/event/FMLEventHandler.java

```diff
@@ -5,7 +5,9 @@
 import com.minecolonies.coremod.colony.ColonyManager;
 import com.minecolonies.coremod.network.messages.ColonyStylesMessage;
 import com.minecolonies.coremod.network.messages.ServerUUIDMessage;
+import com.minecolonies.coremod.sounds.ModSoundEvents;
 import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.util.SoundEvent;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.util.SoundEvent'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AUselessImportCheck)